src/fullscreen button.js

```diff
@@ -0,0 +1,53 @@
+/**
+🖥️
+@file fullscreen button
+@summary Adds a fullscreen button to the lower-right of the game canvas.
+@license MIT
+@author Violgamba (Jon Heard)
+@version 4.7.0
+
+
+@description
+Adds a fullscreen button to the lower-right of the game canvas.  This is useful if the site your
+game is hosted on does not automatically add fullscreen support.
+
+
+HOW TO USE:
+1. Import this plugin into your game.
+2. Playtest the game.  Note the transparent button on the lower-right of the game window.
+3. Click the transparent button and note that the game goes fullscreen.
+4. Hit the "escape" key and the game becomes non-fullscreen.
+*/
+
+const playerUi = document.getElementById('player');
+const fullscreenButton = document.createElement('button');
+playerUi.append(fullscreenButton);
+fullscreenButton.id = 'btnToggleFullscreen';
+fullscreenButton.setAttribute('style', 'position:absolute;width:21px;height:21px;right:2px;bottom:2px;padding:0;opacity:40%;');
+fullscreenButton.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" stroke="currentColor" stroke-linecap="round" fill="none" stroke-width="1.25" width="14" height="14"><path d="M5 2h-3v3"></path><path d="M11 2h3v3"></path><path d="M2 11v3h3"></path><path d="M14 11v3h-3"></path></svg>`;
```

Comment:
imo the icon + style here should be configurable, and the way you've styled this leaves most of the default styling intact, which will make it appear differently on different browsers. i'd recommend injecting a stylesheet and including more deliberate styles (including hover/focus state)